etl/tube_lite

```diff
@@ -627,24 +529,37 @@ def etl(credentials_path, endpoint, output_path, batch_size, elastic):
 
     # assumes guppy-setup dropped ES indices
     #_writer(drop_file_indexes(elastic))
+
     # write data
 
-    # for i, row in enumerate(read_files(sc, batch_size)):
-    #     if i == 0:
-    #         _writer(create_file_indexes(row, elastic))
-    #     _writer(write_file_data(row, elastic))
-    # _writer(write_file_array_config(elastic))
-    # _writer(write_file_alias_config(elastic))
-    # _writer(write_file_array_config(elastic))
+    print(f'Reading files. batch_size {batch_size}')
+    for i, row in enumerate(read_files(sc, batch_size)):
+        if i == 0:
+            print('Creating file indices.')
+            _writer(create_file_indexes(row, elastic))
+        _writer(write_file_data(row, elastic))
+        if i % batch_size == 0:
+            print(f'Wrote {i} files')
+        if limit and i > limit:
+            break  # for testing
+    _writer(write_file_array_config(elastic))
+    _writer(write_file_alias_config(elastic))
+    _writer(write_file_array_config(elastic))
```

Comment:
Should line 547 be `_writer(write_array_aliases(elastic))` like on [line 650](https://github.com/ACED-IDP/compose-services-training/pull/18/files#diff-101db346d31bd8dca6453074199dbc629a016e63a5ce0f77eef9759ea333ae21L650) below?

Reply:
will check.  thanks